File: src/Atividade1/Exercicio 3.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>

int main()
{
    int A[10],i,menor;

    for (i=0; i<10; i++)
    {
        printf("A[%d] =",i+1);
        scanf("%d",&A[i]);
    }
    menor = A[0];
    for (i=0; i<9; i++)
    {
        if(menor>A[i+1])
        {
            menor=A[i+1];
        }

    }
    printf("Menor valor = %d", menor);
}

File: src/Atividade2/Exercicio3.java
package Atividade2;

import java.util.Scanner;

public class Exercicio3 {
    public static void main(String[] args) {
       int[] vetor = new int[10];
       int menor, posMenor=0;
        Scanner leitor = new Scanner(System.in);
        System.out.println("Digite os valores:");
        for (int i=0; i<10; i++){
            System.out.print("Vetor [" + (i+1) + "]:");
            vetor[i]=leitor.nextInt();
        }
        menor = vetor[0];
        for (int i=0; i<10; i++){
            if(menor>vetor[i]){
                posMenor = i;
                menor = vetor[i];
            }
        }
        System.out.println("Menor valor: " + menor);
        System.out.println("Posicao do menor valor: " + (posMenor+1));
    }
}
